### DISCUSSION TITLE
BlackHole 64ch keeps setting the primary volume slider to -44dB

### DISCUSSION THREAD

**Discussion by NiccoloGranieri** [ExistentialAudio/BlackHole#741] [Q&A]
Hi everyone, first time poster so sorry if I'm breaking any rules.

I am using all three Blackholes for an audio setup, and only the 64ch one keeps setting the primary volume slider to -44dB. Once I correct it, it stays as is, but as soon as I restart the laptop it goes back to -44.

Anyone know how I could fix this? Thanks!

**Comment by devinroth** [comment]
No rules broken.

All the different channel versions use the exact same code except changing the name and channel count.

Do a test. Set all 3 BlackHole volumes to something specific, restart and see which ones change. They are suppose to save the last volume level and load it on restart. Actually you can use the terminal command to reboot CoreAudio with the same affect.

Chances are there’s an audio program that is doing it and not BlackHole.

Devin

**Comment by NiccoloGranieri** [comment]
Hi Devin,

thanks so much for replying.

Turns out BH2 and 16 behave correctly, but BH64 likes going back to -44. I'll do some investigation, but I wouldn't know where to start to be honest!

**Comment by devinroth** [comment]
Make sure you have no audio applications running and test again.

I’ll see if I can reproduce on my side.

Devin

**Comment by NiccoloGranieri** [comment]
Hi Devin,

sorry for the late reply. No audio apps open. Moreover, reinstalling the app not only doesn't fix the issue, but BH64 gets set to -44 as soon as the installation is finished.

**Comment by devinroth** [comment]
Can’t reproduce. I did a clean install of the 64 version and it was set to 0db. I move to a different value, restarted core audio and it remained there. Restarted the computer and it reset to 0db. So I have no idea what’s happening on your machine.

Do you have another machine you can test on?

Devin

**Comment by NiccoloGranieri** [comment]
Hi @devinroth, can't test on another machine at the moment, but can try to find another arm mac and see what's happening there.

If it helps, I just restarted my mac in safe mode, and tried the move slider, `sudo killall coreaudiod`, check slider combo. Unfortunately, the problem persists.
